src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -991,6 +991,14 @@ export default {
       }
     },
 
+    /** Returns the computed filing author. */
+    filingAuthor (filing: FilingIF): string {
+      const header = filing.header
+      if (header.isStaff) return 'Registry Staff'
```

Comment:
Is this a new(er) property in the header?

Reply:
Yes. Unless Thor tells me something different.